main.tex

```diff
@@ -708,21 +752,21 @@
 
 \item  stored on Common Property except in areas designated by the Owners Corporation or otherwise with its written consent; and
 
-\item  brought into a Lot (other than a Lot designated for parking) or the foyer, lifts, stairwells, hallways, garden areas, walkways, balconies or other Common Property.
```

Comment:
You are currently banned from taking a bicycle to your apartment, if you went through a hallway.